Fix implicit declaration of functions min_state, min_state_alt

https://github.com/nagios-plugins/nagios-plugins/issues/582
https://github.com/nagios-plugins/nagios-plugins/pull/583
--- plugins/utils.h.orig
+++ plugins/utils.h
@@ -81,6 +81,8 @@ int xasprintf (char **strp, const char *fmt, ...);
 
 int max_state (int a, int b);
 int max_state_alt (int a, int b);
+int min_state (int a, int b);
+int min_state_alt (int a, int b);
 
 void usage (const char *) __attribute__((noreturn));
 void usage2(const char *, const char *) __attribute__((noreturn));
